### PR TITLE
add hint that these files are managed by modulesync

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: moduleroot/.puppet-lint.rc.erb [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Managed by modulesync - DO NOT EDIT
```
Comment (rwaffen) [inline moduleroot/.puppet-lint.rc.erb:1]: here i'am not 100% sure if this works like this

Comment (bastelfreak) [reply moduleroot/.puppet-lint.rc.erb:1]: can you test that please?

Comment (rwaffen) [reply moduleroot/.puppet-lint.rc.erb:1]: yes, will do

Comment (rwaffen) [reply moduleroot/.puppet-lint.rc.erb:1]: seems to work, does not cause an error or change the output

**Review by smortex — APPROVED** [review]
Looks reasonable!

**Pull request merged**